src/constants.js

```diff
@@ -1,11 +1,16 @@
+const packageInfos = require('../package.json')
 const ADD_COZY_METADATA = true
-const APP_NAME = 'konnector-google'
+const APP_NAME = packageInfos.name
```

Comment:
Do we want the name from `package.json` or the name from the manifest here? (I did it in cozy-contacts https://github.com/cozy/cozy-contacts/blob/master/src/helpers/contacts.js#L3 but forgot to do it here)

Reply:
Aaah I knew I had seen this somewhere but couldn't find where! I think using the manifest makes more sense, I'll update the PR 👍